openstack/castellum/templates/secret.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: castellum-secret
+data:
+  service_user_password: {{ required ".Values.castellum.service_user.password" .Values.castellum.service_user.password | b64enc }}
+  postgres_password: {{ required ".Values.postgresql.postgresPassword" .Values.postgresql.postgresPassword | b64enc }}
+  rabbitmq_password: {{ required ".Values.castellum.rabbitmq.password" .Values.castellum.rabbitmq.password | b64enc }}
```

Audit logging is optional in the `values.yaml`, so this value should not be `required` here:
```suggestion
  rabbitmq_password: {{ .Values.castellum.rabbitmq.password | b64enc }}
```